src/app/register/register.component.html

```diff
@@ -0,0 +1,98 @@
+<div class="container my-5">
+    <section class="row">
+        <div class="col-lg-6">
+            <h2>Signup</h2>
+            <p>We do not share your personal details with anyone.</p>
+        </div>
+
+        <div class="col-lg-6">
+            <form [formGroup]="registerForm" novalidate (ngSubmit)="onSubmit()">
+                <div class="form-group custom-group">
+                    <input type="text" id="firstName" class="form-control" formControlName="firstName" placeholder=""
+                        required aria-label="Enter firstName">
+                    <label for="firstName" class="floating-label">First Name</label>
+
+                    <div *ngIf="registerForm.controls['firstName'].touched && !registerForm.controls['firstName'].valid"
+                        class="alert alert-danger">
+                        <div *ngIf="registerForm.controls['firstName'].hasError('required')">
+                            Please enter first name
+                        </div>
+                        <div *ngIf="registerForm.controls['firstName'].hasError('minlength')">
+                            Please enter atleast 4 characters
+                        </div>
+                    </div>
+                </div>
+
+                <div class="form-group custom-group">
+                    <input class="form-control" type="text" formControlName="lastName" id="lastName" placeholder=""
+                        required aria-label="Enter lastname">
+                    <label for="lastName" class="floating-label">Last Name</label>
+
+                    <div *ngIf="registerForm.controls['lastName'].touched && !registerForm.controls['lastName'].valid"
+                        class="alert alert-danger">
+                        <div *ngIf="registerForm.controls['lastName'].hasError('required')">
+                            Please enter last name
+                        </div>
+                        <div *ngIf="registerForm.controls['lastName'].hasError('minlength')">
+                            Please enter atleast 4 characters
+                        </div>
+                    </div>
+                </div>
+
+                <div class="form-group custom-group">
+                    <input class="form-control" type="text" formControlName="email" id="email" placeholder=""
+                        pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required aria-label="Enter email">
```

Comment:
Please consider the same comments as the `login` component for this component too